CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ NOTE: FreeBSD builds will not be included for this release. There is a bug in an
 upstream library preventing cross-compilation of the Grafana Cloud Agent for
 this platform. FreeBSD builds will return in a future release.
 
+# v0.6.0 (2020-09-04)
```

Comment:
Should this Note be duplicate above?

Reply:
Yeah, mostly as a reminder until I fix FreeBSD builds